gateway/api_loader.go

```diff
@@ -679,7 +680,7 @@ func loadHTTPService(spec *APISpec, apisByListen map[string]int, gs *generalStor
 		router.Handle(chainObj.RateLimitPath, chainObj.RateLimitChain)
 	}
 
-	router.Handle(chainObj.ListenOn, chainObj.ThisHandler)
+	subrouter.NewRoute().Handler(chainObj.ThisHandler)
```

Comment:
Is it equal to `subrouter.Handle("/", chainObj.ThisHandler)`? 
And if not, what is the difference?

Reply:
It is not equal. The difference is `/` only matches `/`. However, the `NewRoute()` matches anything.
With `/` you will get `Not Found` for a simple httpbin get operation.